### ISSUE TITLE
[Docs] Add Docker COPY from Example

### ISSUE THREAD

**Issue by yordis** [pgvector/pgvector#497]
Hey there, I am aware of https://github.com/pgvector/pgvector/issues/484, but I am asking something differently,

We could ["merge"](https://github.com/moby/moby/issues/3378#issuecomment-411824851) the `pgvector` image into the base image.

- Why would you do that?

I have other extension requirements aside from `pgvector`, and I am hoping that such a pattern becomes familiar enough so Extension Authors could provide a straightforward image to `COPY` from instead of relying on tools like `pgxn` or `pgxman`, or deal with "complex" (unfamiliar) installations steps and instead take advantage of Official Docker Image.

```dockerfile
FROM postgres:latest

COPY --from=pgvector/pgvector:pg16 /[help here] /[help here]
```

I appreciate any help you can provide.

**Comment by ankane** [comment]
Hi @yordis, feel free to share if you get this working. You can see which files pgvector installs by looking at the output of `make install`.

**Comment by yordis** [comment]
@ankane, the intent is to add official documentation for it, more than me figuring it out. The community needs some alignment and docs that simplify the situation.

I would ask you to reconsider opening the issue and taking the perspective of non-postgres Operational experts.

I will figure it out personally, but it is more than just that.

**Comment by jkatz** [comment]
If this is targeted for a production build, I'd recommend just installing pgvector from a package. I checked the upstream PostgreSQL repos, and both yum and apt include pgvector.

**Comment by yordis** [comment]
@jkatz I ended up just using `apt`, but I wish it happened across the ecosystem: to be able to pull from another Docker Image into the base one. Not everyone is publishing, but I found they mostly provide Docker images.

Imagine the following as a standard:

```dockerfile
FROM postgres:16-bookworm

COPY --from=pgvector/pgvector:1.5.0-extension /PGEXTENSION/${PG_MAJOR} /
````

That tree under `/PGEXTENSION` has ONLY the files required to copy into my image, making it safe to copy everything from it into the `/` safely.
